src/components/card/demoBasicUsage/index.html

```diff
@@ -4,7 +4,7 @@
   <md-content>
 
     <md-card>
-      <img src="img/washedout.png" class="md-card-image" alt="Washed Out">
+      <img src="{{imagePath}}" class="md-card-image" alt="Washed Out">
```

Comment:
Use `ng-src` (here and below). Doing it this way will make the browser fire an HTTP request off for `"{{imagePath}}"`